src/android/java/com/mattrayner/vuforia/app/ImageTargets.java

```diff
@@ -57,40 +57,41 @@
 public class ImageTargets extends Activity implements ApplicationControl
 {
     private static final String LOGTAG = "ImageTargets";
-    
+    private static final String FILE_PROTOCOLE_FIX = "file://";
+
```

Comment:
Hey @Stending, it's fine to merge actually; we're gonna be releasing all the new updates as 2.0 today. Thanks for this!